apps/synapse-portal-framework/src/components/crc-researcher/ParticipantsBarPlot.tsx

```diff
@@ -1,18 +1,18 @@
-import { CSSProperties, useEffect, useState } from 'react'
+import {
+  QueryBundleRequest,
+  QueryResultBundle,
+  RowSet,
+} from '@sage-bionetworks/synapse-types'
+import cloneDeep from 'lodash-es/cloneDeep'
 import Plotly from 'plotly.js-basic-dist'
-import _ from 'lodash-es'
```

Comment:
Not sure why upgrading TypeScript cause using the default export from lodash-es to break, but it's a good thing, because this import is hard to tree-shake